projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/package.json

```diff
@@ -115,6 +114,9 @@
     "outputDir": "vdk-jupyterlab-extension/labextension",
     "schemaDir": "schema"
   },
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
May you comment, why setting the compiler option?

Reply:
It should be deleted. I was experimenting with something and forgot to delete it. Will be deleted.